src/wrapper.jl

```diff
@@ -106,22 +214,952 @@ const FT_PURGE_TX = 2
   FT_OTHER_ERROR,
   FT_DEVICE_LIST_NOT_READY)
 
-@enum(
-  FTWordLength,
-  BITS_8 = FT_BITS_8,
-  BITS_7 = FT_BITS_7)
+# Types
+# 
 
-@enum(
-  FTStopBits,
-  STOP_BITS_1 = FT_STOP_BITS_1,
-  STOP_BITS_2 = FT_STOP_BITS_2)
+# FT_GetDeviceInfoList Struct
+struct FT_DEVICE_LIST_INFO_NODE
+  flags::ULONG
+  typ::ULONG
+  id::ULONG
+  locid::DWORD
+  serialnumber::NTuple{16, Cchar}
```

Comment:
Why is this a tuple of Chars, seems very strange

Reply:
This is a struct which is passed directly into `FT_GetDeviceInfoList`. The `serialnumber` and `description` within that struct should be pre-allocated `char*` arrays. See the documentation on [struct type correspondences](https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/#Struct-Type-correspondences-1).

Reply:
I've added a note in 17a99f24f4996743880a1f61c4eb5ef5d87aa32e

Reply:
So the C side of this is a `char bla[N]`?

Reply:
```julia
struct FT_DEVICE_LIST_INFO_NODE
   ...
   serialnumber::NTuple{16, Cchar}
   ...
end
```
is 
```C
typedef struct _ft_device_list_info_node {
   ...
   char SerialNumber[16];
   ...
} FT_DEVICE_LIST_INFO_NODE
```